ffi/freetype.lua

```diff
@@ -125,10 +125,46 @@ function FTFace_mt.__index:getHeightAndAscender()
 end
 
 function FTFace_mt.__index:done()
+    -- Note that past this point, accessing self is illegal, FT2 frees the pointer
     assert(ft2.FT_Done_Face(self) == 0, "freetype error when freeing face")
 end
 
-FTFace_mt.__gc = FTFace_mt.__index.done;
```

Comment:
No more need for this `__gc` ? We should not forget to explicitely call `done()` when done ? :)

Reply:
It never worked properly (and be glad, as it would lead to a double-free). __gc is used only for types that do ffi.new() - which we don't (we ffi new its pointer box, different type).